### DISCUSSION TITLE
graphql/codegen/plugins/typescript/typescript-nhost

### DISCUSSION THREAD

**Discussion by giscus** [dotansimha/graphql-code-generator#9711] [Docs Discussions]
# graphql/codegen/plugins/typescript/typescript-nhost

https://the-guild.dev/graphql/codegen/plugins/typescript/typescript-nhost

**Comment by kelkes** [comment]
Doesn't work anymore. NhostClient doesn't have a schema property anymore.

**Reply by fourcolors** [reply]
was just trying to use this and found this out as well. Is there a work around?
